wiki/en/Running-a-Server.md

```diff
@@ -36,86 +37,108 @@ However, various problems can also arise when setting up Servers - especially wh
+For information, [see the Directories guide](Directories).
 
-If you want to run a Server on a Raspberry Pi (or a different armhf/arm64 debian-based device), you will need to download the `.deb` files for 32 bit `armhf` or 64 bit `arm64`, not the default `amd64` ones you'd use on an Intel/AMD based machine.
+---
 
+## Configuration options
 
-### Installation
+Depending on your operating system and how you are running the server, you can set Server options and make them persistent between reloads as follows:
 
-1. Download the [latest headless (amd64) .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }}) or, if you use a Raspberry Pi etc. download the [latest armhf .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }}) or the [latest arm64 .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-arm64 }})
-1. Update apt to make sure you have a current list of standard packages: `sudo apt update`
-1. Install the Jamulus package: `sudo apt install ./{{ site.download_file_names.deb-headless }}` or for RasPi etc. armhf: `sudo apt install ./{{ site.download_file_names.deb-headless-armhf }}`; arm64: `sudo apt install ./{{ site.download_file_names.deb-headless-arm64 }}`
-1. Enable the headless Server process via systemd: `sudo systemctl enable jamulus-headless`
-1. Add your desired [command line options](Running-a-Server#command-line-options) to the `ExecStart` line in the systemd service file by running `sudo systemctl edit --full jamulus-headless` (By default you will be running an Unregistered Server).
-1. Reload the systemd files `sudo systemctl daemon-reload` and restart the headless Server: `sudo systemctl restart jamulus-headless`
-1. Check all is well with `systemctl status jamulus-headless` (hit `q` to get back to the command prompt).
+**For Linux headless** (Debian/Ubuntu using systemd) 
 
-You can control Jamulus with the `systemctl` command. For example, to stop the Server cleanly:
+Add your desired command line options to the `ExecStart` line in the systemd service file by running `sudo systemctl edit --full jamulus-headless` (You will need to reload the restart for the changes to take effect. See [Installation](#installation))
 
-`sudo systemctl stop jamulus-headless`
+**For the GUI** (all platforms)
 
-_To upgrade your Server, just repeat the steps above._
+Any settings made using the graphical interface will be stored in the `Jamulusserver.ini` file (do not attempt to edit this file by hand). However, some options are not available via the GUI and need to be set via the command line. For more information, see [Servers on the desktop](#servers-on-the-desktop)
 
-### Configuration
+---
 
-#### Running in Registered mode
+### Server mode-related options
 
-The following minimum setup is required to [run a Registered Server](Running-a-Server#server-types):
+##### `-e or --directoryserver` 
 
-~~~
-jamulus --nogui --server \
```

Comment:
Previous steps install and start **jamulus-headless**. This section specifies **jamulus**. But it looks like your changes only speak in terms of jamulus-headless, and controlling it through systemctl. Good. Except I used this little snippet with these four flags to spin up a server using screen. This snippet got me running fast! Removing it, am I asked to gather up the four CLI --settings involved, and construct the command line within `sudo systemctl edit --full jamulus-headless` ? I think installation should try to get the user running as simply as possible. We have steps that get an UNregistered server going. We should then immediately explain how to make it a registered server. That's what this snippet did for me, so I'm reluctant to give it up.

Reply:
We're talking about Linux, so yes there's going to be more than one way of doing things :-) This is why the "Configuration options" section start with the words "_Depending on your operating system and how you are running the server, you can set Server options and make them persistent between reloads as follows:_"

The command in question may get you "running as simply as possible", but without understanding what you're doing with it, it's not going to make your life easier :-)

There is a custom with some docs to do have a "quick install" section (eg see Wordpress) which assumes people know what they're doing. The question then is how much info to have without it being a waste of time if (say)  they triumphantly spin up a server and then wonder why nobody can connect to it.

BTW for that matter, there is the school of thought that Jamulus should **only** document directly related Jamulus things - so we'd never mention stuff like `systemctl edit` how to start/stop recording with bash scripts, or even how to install the binary, because we'd assume those things are not worth re-explaining. Something to consider!

Reply:
My impression from this change is: "We'll get you set up with an unregistered server easy-peasy, but if you want a registered server, sit down and study for a while." We should provide the quick-and-easiest steps to standing up a _registered_ server, which starts with steps for unregistered and then adds a little bit, in step sequence. Maybe that uses systemctl rather than jamulus-headless with those 4 flags, but it results in simplest steps for the people who may not know what they're doing.

Reply:
We could have a section/page called "Server Installation" or similar that provides the steps to get each of the three modes running. But without knowing what the difference is between, say, an unregistered server and a Directory, knowing how to get one running is a bit pointless. For that you need to read the manual. This is why I've structured this PR in the way I have.

BTW what do you mean by "systemctl rather than jamulus-headless with those 4 flags"?

Reply:
I want the steps to adding a _registered_ server to be nearly mindless, with an extra sentence to explain how to pick a --directoryserver value.  People don't need to know what a Directory server is, and every learning requirement increases friction. If I can't persuade you, that's alright. Later I'll make a PR hill I'm willing to die on, called QuickStart Hill.

I'm still running two servers using line 108-111. I believe the draft only uses systemctl to control the server, which might be fine.

Reply:
As I said in an earlier comment, there is the option of adding a "quick start" section, but I don't think there'd be much in it (and it would be in danger of repeating information, which repetition we need to avoid for maintenance reasons). And it stands to reason that if you already know what the difference is (and requirements are) between a registered and unregistered Server, then pretty much by definition you won't need a quick start, no? But feel free to give it a go.

On the systemctl thing - we only really support Jamulus headless for systemd based distros, so this is why we use the recommended systemd way of handling Jamulus as a service.

Reply:
Repeating a sentence or two isn't a danger. I'm my first user here: I want to follow a sequence that results in a running server. And I want everyone to be able to do this with minimal thinking. I bet this draft will reduce server adoption because it adds a study session.

Reply:
> Repeating a sentence or two isn't a danger. 

It depends what the sentence says :-) 

> And I want everyone to be able to do this with minimal thinking. 

Well, as I say - we can do that (in another PR), but it seems to be a bit of waste of time if it results in people running things they don't properly understand.

Reply:
I improperly understood directories until about last week. The purpose of engineering is to hide complexity.

Reply:
Were you administering a Directory?

Reply:
Like 99% of the userbase, I probably never will.

Reply:
In which case, an admin guide isn't what you'd read.

Reply:
Yes, as noted earlier, I think we'll probably change to "Running a Server".

Reply:
In which case something like

----

## Quick Start
(For simplicity, start with a Server registered with a built in Directory.  Link to elsewhere in the guide for other configurations.  Same for all platforms.)

### Linux on Debian/Ubuntu using the headless .deb

#### Assumptions
...

### Linux Other
...

### Windows
...

### MacOS
...

## Other sections

## ...

----

might well be useful.

Reply:
There's no material difference between each OS other than headless Linux though (which is just the apt and systemctl commands really). "_Download the installer, run it and double-click the Server icon. Now read the rest of the guide for information about configuration_" doesn't seem to solve a significant problem. Can you provide a bit more detail about what you mean?

Reply:
What significant problem are you referring to?

I'd like the shortest path to get a new server onto a public directory. The current publication provides bash code that lets me enter a --directoryserver address that I pick from a table. This PR will require studying the Server Modes sections. I want a QuickStart to conclude before the deep dive begins. Losing lines 108-110 will make me sad. I don't want to study that list of flags. I wanna paste in the minimum!

Reply:
> What significant problem are you referring to?

I don't know - that's why I'm asking.

> I'd like the shortest path to get a new headless Linux server onto a public directory.

That's becuase you know what getting a server onto a public directory means. But for somone who doesn't, there's not a lot of point in a fast track to something they don't understand. 

Do you mean you want a section for people who have already undertood the differences between server modes and the various requirements like port forwarding etc., and just want to be reminded how to run one?

Reply:
I absolutely do want someone who "doesn't know what getting a server onto a public directory means" to succeed at the task anyway. In fact, we ask them to do so:  "It is highly recommended to test your Server by registering it on one of the built-in Directories first so as to narrow down any subsequent problems in unregistered mode." 

The QuickStart concept might seem odd because it actually tries to explain very little, in contrast to most documentation. [Here's a QuickStart I wrote](https://learn.microsoft.com/en-us/azure/cognitive-services/speech-service/get-started-speech-to-text?tabs=windows%2Cterminal&pivots=programming-language-cli). It's brief! I just want them to press the right buttons and be up and running! Adoption, rather than deep understanding, is the goal.

Reply:
I think this is an issue for a separate PR after this one is merged. Marking as resolved for now.